reference/import_export/index.rst

```diff
@@ -5,20 +5,16 @@ This chapter describes the architecture and behaviour of imports and exports in
 
 .. note::
 
-  The import part has been widely re-worked in 1.4.
-
-  A new system has been introduced, the old system has been deprecated but is kept, both systems are usable in 1.4 but we strongly recommend using the new one.
-
-  The new system is composed of a new Connector Bundle, a new Connector Component and some new Catalog APIs.
-  The deprecated system consists of parts of the BaseConnector Bundle and parts of the Transform Bundle (we aim to depreciate entirely these bundles).
+  The import part has been widely re-worked in 1.4: a new system has been introduced, and the old one is deleted since 1.6 version.
```

Comment:
+  The import part has been widely re-worked in 1.4: a new system has been introduced, and the old one has been kept deprecated during 2 minor versions (1.4 and 1.5) and is now deleted since 1.6 version.